src/main/java/Network.java

```diff
@@ -17,12 +20,19 @@ public void start() {
     }
 
     private void performLearning() {
-        final int eachIteration = 50;
-        network.addListeners(new ScoreIterationListener(eachIteration));
+        CollectScoresIterationListener sc= new CollectScoresIterationListener();
```

Comment:
What's the purpose of CollectScoresIterationListener vs ScoreIterationListener?

Reply:
CollectScoresIterationListener has function exportScores(File f)